### DISCUSSION TITLE
PPOCR-V4是否支持多点标注的数据集？

### DISCUSSION THREAD

**Discussion by xaclincoln** [PaddlePaddle/PaddleOCR#12384] [Q&A] [answered]
看了文档中关于准备数据的说明，数据标注的范例是给出了4点标注。当前有个场景是有不少弯曲文本，按照DbNet或者DbNet++的论文看，应该是支持多点标注的。看了issue中老早之前有关多点标注标注的问题，有人回答不支持，想再次确认下现在（2024/01/17）PPOCR-V4是否能够支持自定义数据集上的弯曲文本检测？

**Comment by tink2123** [comment, marked as answer]
PPOCR-V4 不支持，目前还是四点框。 repo 中提供了DBNet++的实现，可以参考；https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/algorithm_det_db.md
